pyccel/codegen/printing/cucode.py

```diff
@@ -0,0 +1,121 @@
+# coding: utf-8
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+"""
+Provide tools for generating and handling CUDA code.
+This module is designed to interface Pyccel's Abstract Syntax Tree (AST) with CUDA,
+enabling the direct translation of high-level Pyccel expressions into CUDA code.
+"""
+
+from pyccel.codegen.printing.ccode import CCodePrinter, c_imports
+
+from pyccel.ast.variable    import InhomogeneousTupleVariable
+from pyccel.ast.core        import Declare, Import, Module
+
+from pyccel.errors.errors   import Errors
+
+
+import_dict = {'omp_lib' : 'omp' }
+
+c_library_headers = (
+    "complex",
```

Comment:
Is this dictionary necessary? Can you not use the inheritance for this?

Reply:
[fc69c53](https://github.com/pyccel/pyccel-cuda/pull/32/commits/fc69c53e9792d21162922cc71928ccf5f1195e47)